src/layouts/default.vue

```diff
@@ -137,7 +137,8 @@ export default {
     grid-area: body;
     grid-template-rows: auto 1fr;
     padding: 0 20px 20px 20px;
-    overflow: auto;
+    overflow-y: auto;
```

Comment:
`overflow-y: auto` should be the default already. (Though I guess it's okay to be explicit.)